using_images/other_images/jenkins.adoc

```diff
@@ -111,58 +119,73 @@ For the `edit` `*Role*`, in addition to the permissions available to `*view*`:
+administrative privileges will not be given those privileges when
+{product-title} OAuth is used, unless the {product-title} cluster administrator
+explicitly defines that user in the {product-title} identity provider and
+assigns the `admin` role to the user.
 ====
 
-Permissions for users in Jenkins can be changed in {product-title} after those users are initially established in Jenkins.  The OpenShift Login plugin polls the {product-title} API server for permissions and will update the permissions stored in
-Jenkins for each Jenkins user with the permissions retrieved from {product-title}.  If the Jenkins UI is used to update permissions for a Jenkins user, the permission changes will be overwritten the next time the plugin polls {product-title}.
+Jenkins' users permissions can be changed after the users are initially
+established. The OpenShift Login plug-in polls the {product-title} API server
+for permissions and updates the permissions stored in Jenkins for each user with
+the permissions retrieved from {product-title}. If the Jenkins UI is used to
+update permissions for a Jenkins user, the permission changes are overwritten
+the next time the plug-in polls {product-title}.
+
+You can control how often the polling occurs with the
+`OPENSHIFT_PERMISSIONS_POLL_INTERVAL` environment variable. The default polling
+interval is five minutes.
 
-You can control how often the polling occurs with the `OPENSHIFT_PERMISSIONS_POLL_INTERVAL` environment variable.  The default polling interval when no environment variable is set is 5 minutes.
+.Creating a new Jenkins pod
 
-The following commands creates a new Jenkins xref:../../architecture/core_concepts/pods_and_services.adoc#pods[pod] using {product-title} OAuth authentication,
-assuming you have installed xref:../../install_config/imagestreams_templates.adoc[the default imagestreams and templates].
+. Ensure you have installed
+xref:../../install_config/imagestreams_templates.adoc[the default imagestreams
+and templates].
 
-Run this command to use persistent volumes:
+. Then create a new Jenkins application using:
+.. Persistent volumes:
 ----
 $ oc new-app jenkins-persistent
 ----
 
-Run this command to use an EmptyDir type volume (configuration will not persist across pod restarts):
+.. Or an EmptyDir type volume (where configuration does not persist across pod restarts):
 ----
 $ oc new-app jenkins-ephemeral
 ----
 
 [NOTE]
 ====
-If you instantiate the template against releases prior to v3.4 of {product-title}, standard Jenkins authentication will be used.  The default 'admin' account will exist with password 'password'.
-See Jenkins standard authentication for details about changing this password.
+If you instantiate the template against releases prior to v3.4 of
+{product-title}, standard Jenkins authentication is used, and the default
+'admin' account will exist with password 'password'. See Jenkins standard
+authentication for details about changing this password.
```

Comment:
Even though the  referenced section is directly below this, it would help to add a link to the topic. There is always the possibility of content getting moved around or expanded.